playbook.yml

```diff
@@ -76,6 +76,12 @@
     cron:
       name: Synchronize WPT
       job: "cd /root/web-platform-tests/ && git pull origin master && systemctl restart wpt"
+      minute: '*'
+
+  - name: Add cron job to restart wpt once a day to get any changes to wptserve
```

Comment:
This obviates the other cron job, doesn't it?

Reply:
Oops I forgot to remove the restart from the other cron job.